custom/ewsghana/reports/specific_reports/stock_status_report.py

```diff
@@ -73,21 +73,27 @@ def charts(self):
             def convert_product_data_to_stack_chart(rows, chart_config):
                 ret_data = []
                 for k in ['Stocked out', 'Not Stocked out', 'No Stock Data']:
+
+                    def calculate_percent(x, y):
+                        return float(x) / float((y or 1))
```

Comment:
Could `y` ever be `'0'` or some equivalent string? If yes, this will throw an error.

Would it make sense to return an empty string, or `_("Divide by zero")` if `y` evaluates to zero?